orte/mca/errmgr/default_hnp/errmgr_default_hnp.c

```diff
@@ -349,6 +487,7 @@ static void job_errors(int fd, short args, void *cbdata)
 
 static void proc_errors(int fd, short args, void *cbdata)
 {
+    orte_propagate.register_cb();
```

Comment:
What is this ?

Reply:
Register propagator message receive callback. 
I want reliable broadcast in grpcomm be general - which means it only broadcast/receive,  once receive a message , it doesn't care what in the message and give the message back to propagator.